rai/client.go

```diff
@@ -1713,3 +1713,65 @@ func (c *Client) ListSnowflakeDatabaseLinks(
 	}
 	return result, nil
 }
+
+//
+// Snowflake Data Streams
+//
+
+func (c *Client) CreateSnowflakeDataStream(
```

Comment:
isView is more of an option, I wonder if it would be better to put some of these args into a struct

Reply:
For no. 2 perhaps we can at least name it `fullObjectName` or something, on top of adding validation so it fails early if it's not fully qualified and adding an example in `examples/`?

Reply:
the db link db and db link schema are not necessarily the same as the db and schema that's part of the object's fq name. example from wien:

> in the sandbox db, i put the dblink in the RAI schema. the objects from this dblink are then used targeting objects in other schemas in the sandbox db

Reply:
Ok here's the question that pertains to our model itself to answer: what uniquely identifies a db link? The arguments are (db, schema) but I see what you meant about the schema just being where we store it, rather than an identifying characteristic.  This imples to me that (db) identifies the db link within the scope of an integration, so the DB of the FQN of the object should still identify it right?

Reply:
Answering my own question, the service code creating a db link defines it as 'db-schema', so this is kind of a confusion I'd like to resolve today in standup.  Either 'schema' is a characteristic or its just where we store the RAI code...What does it mean to have two DBLinks for the same db on different schemas, does that add any value?

Reply:
maybe the user passed in a db-link as a fully qualified name, then both reference to snowflake resource are fully qualified